src/query/expression/src/values.rs

```diff
@@ -1251,15 +1250,14 @@ impl Column {
         }
     }
 
-    pub fn from_arrow(
+    pub fn from_arrow_by_array_type(
```

Comment:
```suggestion
    pub fn from_arrow_with_array_type(
```

Reply:
It's better to be moved into `pub fn from_arrow` to avoid any other code using it directly.